scribble-lib/scribble/acmart/acmart.tex

```diff
@@ -23,4 +23,6 @@
 % Normally gets re-written by the title macro:
 \newcommand{\SSubtitle}[1]{{\bf #1}}
 
-
+% Use ACM color; it would be better to use `citecolor` here somehow,
+% but I can't figure out how to do that
+\newcommand{\AutobibLink}[1]{\color{ACMPurple}{#1}}
```

Comment:
Can this take 2 arguments and call `\hyperref` instead of `\color` ?

Reply:
Not in the current design, because the `\hyperref` part is triggered by `link-element`ness, while `\AutobibLink` comes from the style.

Reply:
Ok.
I also couldn't find a way to use `citecolor`